### PR TITLE
Use DOM Future in intro examples instead of outdated DOMRequest (closes #36)

### LINKED ISSUE #36: Intro examples are still using DOMRequest
The introductory examples are using DOMRequest. Should be updated to use Futures.

### PR DESCRIPTION
**Pull request by cdumez**

### CODE REVIEW

File: index.src.html [hunk 1]
```diff
@@ -116,56 +116,54 @@ <h2 id="introduction">
 How to add an alarm the ignores timezone information?
```
Comment (marcoscaceres) [inline index.src.html:116]: Do you want to keep this mention of alarm?

**Comment by marcoscaceres** [comment]
apart from the one thingy above, all looks good :+1:

File: index.src.html (hunk 1 above)
Comment (cdumez) [inline index.src.html:116]: There is a separate bug about making example non-alarm specific. So I left it there for now. Ignoring time zone is really for the Alarm Clock use case anyway.

Comment (marcoscaceres) [inline index.src.html:116]: That's what I thought, but wanted to check.

**Pull request merged**